app/uk/gov/hmrc/eoricommoncomponent/frontend/DateConverter.scala

```diff
@@ -26,11 +29,28 @@ object DateConverter {
 
   private val logger = Logger(this.getClass)
 
+  val earliestYearDateOfBirth         = 1900
+  val earliestYearEffectiveVatDate    = 1970
+  val earliestYearDateOfEstablishment = 1000
+
   def toLocalDate(dateStr: String): Option[LocalDate] =
     Try(LocalDate.parse(dateStr)).recoverWith {
       case NonFatal(e) =>
         logger.warn(s"Could not parse the LocalDate '$dateStr': ${e.getMessage}", e)
         Failure(e)
     }.toOption
 
+  def updateDateOfBirthErrors(errors: Seq[FormError]) = updateYearErrors(errors, earliestYearDateOfBirth)
+
+  def updateDateOfEstablishmentErrors(errors: Seq[FormError]) =
+    updateYearErrors(errors, earliestYearDateOfEstablishment)
+
+  def updateEffectiveVatDateErrors(errors: Seq[FormError]) = updateYearErrors(errors, earliestYearEffectiveVatDate)
```

Comment:
Can you add return types?

I know that in that case it's easy to conclude what is the return type, but it's a good practice to add return types to public methods